client/my-sites/marketplace/pages/submission-success/signup-success.tsx

```diff
@@ -24,17 +29,35 @@ const SignupSuccess = () => {
 			/>
 			<MasterbarLogo />
 			<div className="signup-success">
-				<div className="signup-success__header">
-					<img src={ submissionSuccessImage } alt="Submission success" />
-					<h1 className="signup-success__header-title wp-brand-font">
-						{ translate( 'We’ll be in touch' ) }
-					</h1>
-					<p>
-						{ translate(
-							'Thank you for apply to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.'
-						) }
-					</p>
-				</div>
+				{ currentQuery?.returning !== 'true' ? (
+					<div className="signup-success__header">
+						<img src={ submissionSuccessImage } alt="Submission success" />
+						<h1 className="signup-success__header-title wp-brand-font">
+							{ translate( 'We’ll be in touch' ) }
+						</h1>
+						<p>
+							{ translate(
+								'Thank you for applying to join the WordPress.com marketplace. We’ll be in touch with you very soon to discuss next steps.'
+							) }
+						</p>
+					</div>
+				) : (
+					<div className="signup-success__header">
+						<img src={ submissionSuccessImage } alt="Submission success" />
+						<h1 className="signup-success__header-title wp-brand-font">
+							{ translate( 'Hello, %(username)s', {
+								args: {
+									username: userName,
+								},
+							} ) }
+						</h1>
+						<p>
+							{ translate(
+								'Great! You’re already a vendor! Here is what you can do next while we’re still working on building your Product Dashboard.'
```

Comment:
The double exclamation mark don't look great!

Reply:
Hey @mtias, I created a [new issue](https://github.com/Automattic/wp-calypso/issues/66274) for this change.